Error: Webview API guide: TypeScript error in the "Loading local content" example

The example in the [Loading local content](https://code.visualstudio.com/api/extension-guides/webview#loading-local-content) section contains the following error:

```
src/extension.ts:29:46 - error TS2345: Argument of type 'Uri' is not assignable to parameter of type '"Coding Cat" | "Compiling Cat" | "Testing Cat"'.

panel.webview.html = getWebviewContent(catGifSrc)
                                       ~~~~~~~~~
Found 1 error. Watching for file changes.
```

--- a/api/extension-guides/webview.md
+++ b/api/extension-guides/webview.md
@@ -459,6 +459,20 @@ export function activate(context: vscode.ExtensionContext) {
     })
   );
 }
+
+function getWebviewContent(catGifSrc: vscode.Uri) {
+  return `<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cat Coding</title>
+</head>
+<body>
+    <img src="${catGifSrc}" width="300" />
+</body>
+</html>`;
+}
 ```
 
 If we debug this code, we'd see that the actual value for `catGifSrc` is something like:

